openshift/system/Dockerfile

```diff
@@ -62,7 +63,8 @@ RUN export ${BUNDLER_ENV} >/dev/null \
     && chmod g+w -vfR log tmp public/assets db/sphinx \
     && umask 0002 \
     && cd /opt/system \
-    && npm install --no-progress \
+    && npm install -g yarn \
```

Comment:
If we're ditching `npm`, I think we should install `yarn` directly.

Reply:
Read the description of the PR and the issue attached.